meteor/client/lib/EditAttribute.tsx

```diff
@@ -8,6 +8,7 @@ import { Mongo } from 'meteor/mongo'
 
 import { MultiSelect, MultiSelectEvent } from './multiSelect'
 import { TransformedCollection } from '../../lib/typings/meteor'
+import * as classNames from 'classnames'
```

Comment:
I think this will need to be `import ClassNames from 'classnames'`. This import was changed when the library was updated for meteor 1.10

Reply:
Updated: https://github.com/nrkno/tv-automation-server-core/pull/253/commits/4e6e97434ea45f668f1a4acd8c6b50bdd73458ba